src/auth/externalclient.ts

```diff
@@ -15,7 +15,6 @@
 import {RefreshOptions} from './oauth2client';
 import {
   BaseExternalAccountClient,
-  BaseExternalAccountClientOptions,
   EXTERNAL_ACCOUNT_TYPE,
```

Comment:
because of the suffix `type` I thought this would be an interface or `type`, I would add a comment:

```
// this string constant indicates that external account should  be instantiated ?
```

Reply:
Done. I added clarification here and in `baseexternalclient.ts` where it is declared.